.bash_profile

```diff
@@ -99,3 +99,6 @@ export PATH="$(brew --prefix)/Cellar/sbt@0.13/0.13.18_1/bin:$PATH"
 if [[ -f "$(brew --prefix)/bin/terraform" ]]; then
     complete -C $(brew --prefix)/bin/terraform terraform
 fi
+
+complete -C /opt/homebrew/Cellar/tfenv/3.0.0/versions/1.4.2/terraform terraform
```

Comment:
This line overrides the if statement above,  so should that if statement (and its corresponding code in script/setup) be removed? Or is there a more variable way to do this that will autocomplete with whichever one we wanted, instead of overriding it every time?

Reply:
Good point. Terraform installs to different locations depending on home-brew vs tfenv and intel vs apple chips. At this point, we can probably just assume both apple silicon and tfenv are in use and just use that for completions. Of course, we should probably use the default or highest version in tfenv, rather than just insisting on 1.4.2...

I might break these into 2 or 3 PRs. For the purposes of my current project, the line I really NEED is 

```
export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
```

The rest is just stuff I've had locally but which isn't really required for day to day.

Reply:
Yeah since we're the only ones running this code, and we all have Apple Silicon, that's a safe bet. Besides this everything looks good, so I'll approve it.